README.md

```diff
@@ -65,22 +65,22 @@ and unzip it with your unzip tool of choice if you haven not already done so.
 Finally, open the `qgreenland.qgs` file contained inside with QGIS.
 
 
-## What's Inside the v2.0.0 download package?
+## What's inside the download package?
```

Comment:
Should this be "QGreenland Core package" instead of "download package"?

Reply:
That's how we've referred to it elsewhere, so I would lean toward that.